autoprof/fit/base.py

```diff
@@ -18,14 +19,14 @@ class BaseOptimizer(object):
         relative_tolerance: tolerance for counting success steps as: 0 < (Chi2^2 - Chi1^2)/Chi1^2 < tol [float]
     
     """
-    def __init__(self, model, initial_state = None, relative_tolerance = 1e-3, **kwargs):
+    def __init__(self, model: 'Autorof_Model', initial_state: Any = None, relative_tolerance: float = 1e-3, **kwargs) -> None:
```

Comment:
instead of `Any` for `initial_state` I think the best type hint would be `Sequence`

Reply:
The initial_state comes as a list or tuple?

Reply:
I think the proper type would be `Sequence` since it can be a list, tuple, tensor, ndarray, etc.

Reply:
Okay I'll be changing it...